apps/backoffice-v2/src/common/utils/fetcher/fetcher.ts

```diff
@@ -15,13 +15,21 @@ export const fetcher: IFetcher = async ({
   timeout = 10000,
   schema,
   isBlob = false,
+}: {
+  url: string;
+  method: string;
+  body: BodyInit | null;
+  headers: HeadersInit;
+  timeout: number;
+  options: Partial<RequestInit>;
+  isBlob: boolean;
 }) => {
```

Comment:
`fetcher` receives its types from `IFetcher`. If the types are broken then we need to fix `IFetcher` instead.